### ISSUE TITLE
[META] 🏗4️⃣ test4.gno.land

### ISSUE THREAD

**Issue by moul** [gnolang/gno#362]
Meta issue for https://github.com/gnolang/gno/milestone/4, to keep a place for discussions.
Blocked by #361 

* [ ] initial discord faucet #364 
* [ ] initial namespace support #384
* [ ] fix banker module https://github.com/gnolang/gno/pull/393
* [ ] fix bugs, reorganize content, and prepare the project for the Game Of Realms event.
* [ ] Bonus, this list: https://github.com/gnolang/gno/milestone/4

**Comment by giansalex** [comment]
I share the problems I encountered when launching a testnet with validators

1- It is necessary `stdlibs` directory when starting the node for the first time. (can be packed inside the binary?)
2- node don't search and save other p2p peers, if persistent-peers stop, node stuck
3- `cs.wal` size is larger than the blockchain data (maybe not pruned or reset)
4- no export genesis option

**Comment by zivkovicmilos** [comment]
@moul 
How updated is this issue?

**Comment by moul** [comment]
It's not, let's speak about this during an upcoming team sync.

**Comment by moul** [comment]
Next Steps:

1. Test full node connection and synchronization on `test3`.
2. Create a separate repository for multi-node experimentations to maintain a clean main repository. (will do soon)

**Comment by r3v4s** [comment]
> Meta issue for https://github.com/gnolang/gno/milestone/4, to keep a place for discussions. Blocked by #361
> 
> * [ ]  initial discord faucet [Community faucet bot #364](https://github.com/gnolang/gno/issues/364)
> * [ ]  initial namespace support [feat: r/system/names public functions and checks with AddPackage #384](https://github.com/gnolang/gno/pull/384)
> * [ ]  fix banker module [fix: minting additional GNOTs  #393](https://github.com/gnolang/gno/pull/393)
> * [ ]  fix bugs, reorganize content, and prepare the project for the Game Of Realms event.
> * [ ]  Bonus, this list: https://github.com/gnolang/gno/milestone/4

fix banker module(new pr): #875

**Comment by moul** [comment]
🔗 **Related Proposal:** I've introduced a proposal on a new flavor of testnet that aims to blend the advantages of both staging and testnets. I believe this could address a lot of the concerns and aspirations we have for test4. **[Read the full proposal here.](https://github.com/gnolang/gno/issues/1131)**

Given the potential benefits outlined in the proposal, I'm leaning towards initiating test4 in alignment with this new testnet design. Feedback and further insights are much appreciated.

**Comment by zivkovicmilos** [comment]
@moul 

Can I own this issue / effort?
I want to define with the team the exact scope of `test4` and launch it, given as it's going to be a release candidate for other future, bigger, launches

**Comment by zivkovicmilos** [comment]
`test4.gno.land` launched on July 10th, so this issue is now closed 🚀
